<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1050</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>多功能智慧文字辨識 OCR 2.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>730</y>
      <width>1011</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="button_choose_file">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>選擇檔案</string>
    </property>
   </widget>
   <widget class="Line" name="line_1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>16</width>
      <height>191</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_list_all_file">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>列出所有需要辨識的文件(圖檔/.pdf)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_remove_file">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>移除檔案</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_choose_output_folder">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>輸出位置 (-o)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_output_folder_path">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>370</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>130</y>
      <width>16</width>
      <height>191</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="button_run">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>660</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>執行</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>721</width>
      <height>171</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>C:\Users\junting\Desktop\MODA_company_OCR.pdf</string>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled|ItemIsTristate</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C:\Users\junting\Desktop\1.png</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C:\Users\junting\Desktop\2.png</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C:\Users\junting\Desktop\3.png</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="button_open_output_folder">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>660</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>開啟輸出資料夾</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>760</y>
      <width>981</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>狀態....</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_choose_key_file">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>選擇金鑰檔案</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_keyfile_path">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_keyfile">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Google服務帳戶的金鑰(.json)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_cid">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>430</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_company_name_en">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>490</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_archive">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>560</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>系統自動分類 (-a)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_api">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>560</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>查商業司API (-api)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_rename">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>560</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>檔名重新命名 (-f)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_summary">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>560</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>統計摘要 (-s)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_cid">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>440</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>統一編號 (-c) :</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_company_name_en">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>500</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>公司英文名稱 (-e) :</string>
    </property>
   </widget>
  </widget>
  <action name="actionGithub">
   <property name="text">
    <string>Github</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
